src/Sylius/Behat/Context/Setup/PromotionContext.php

```diff
@@ -95,13 +95,18 @@ public function __construct(
 
     /**
      * @Given there is a promotion :promotionName
+     * @Given there is a promotion :promotionName identified by :promotionCode code
      */
-    public function thereIsPromotion($promotionName)
+    public function thereIsPromotion($promotionName, $promotionCode = null)
```

Comment:
It's generated automatically in ``TestPromotionFactory``, here we give possibility to specify it manually.